modules/sdk-coin-dot/package.json

```diff
@@ -54,6 +54,7 @@
     "hi-base32": "^0.5.1",
     "joi": "^17.4.0",
     "lodash": "^4.17.15",
+    "strip-hex-prefix": "^1.0.0",
```

Comment:
Can we bring this in as a utility? The function is so simple that I don't think we need an npm install to do this.

Reply:
Fixed.